### PR TITLE
Add description about the Chinese Localization SIG

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# sig-chinese-localization
\ No newline at end of file
+# CDF Chinese Localization Special Interest Group
+
+## Group Charter
+
+### Introduction
+
+There's a very big number of developers who is the Chinese speaker. But English is not the first language of them. People prefer to search and read documents in Chinese. Especially for the regular users.
```
Comment (JohnNiang) [inline README.md:7]: First sentence suggestion of this line: There're a very big number of developers who are the Chinese speakers

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+### Purpose
+
+The group focuses on promote projects of CDF in China:
```
Comment (JohnNiang) [inline README.md]: Sentence suggestion: The group focuses on promoting projects of CDF in China

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+* [Daniel](https://github.com/danielfbm) (Alauda)
+* [Jasper](https://github.com/yuezhuangshi) (ZhongYuan Bank)
+* [John Niang](https://github.com/JohnNiang) (QingCloud)
+* [Lan](https://github.com/liangyuanpeng) (Yunhorn)
+* [Shawn](https://github.com/chengjingtao) (Alauda)
+* [Suke](https://github.com/wangsrGit119) (SpringBoot Chinese Community)
+
+## Communication
+
+Communication in this SIG is primarily done through its [Slack channel](hhttps://cdeliveryfdn.slack.com/archives/C028R8G6NRE)
```
Comment (JohnNiang) [inline README.md]: hhttps -> https

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Typos need a cleanup before merging

File: README.md (hunk 1 above)
Comment (LinuxSuRen) [reply README.md:7]: image
image

**Comment by LinuxSuRen** [comment]
> Typos need a cleanup before merging

Thanks for your review. Done with it.

**Comment by yJunS** [comment]
I want to join the members. Please let me join it.
`[Seamus](https://github.com/yJunS) (Jenkins Chinese Community)`

**Comment by robberphex** [comment]
Please add me to the list:

` * [Robert Lu](https://github.com/robberphex) (Robert)`

**Comment by andreafy** [comment]
Please add me to the checklist～

**Comment by LinuxSuRen** [comment]
> Please add me to the checklist～

English name and company (or organization name) is required.

**Comment by LinuxSuRen** [comment]
Considering there's no negative feedback. I'm going to merge this PR. Please file a new PR if anyone wants to improve it.

**Pull request merged**